.github/workflows/build.yml

```diff
@@ -12,13 +12,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: '20'
+          node-version: "lts/*"
```

Comment:
I don't think this is a good idea - we lose any control over what version of Node.js we get. New major versions usually contain breaking changes, so we should update explicitly when we have time to deal with possible issues, rather than being forced to update as soon as a new version comes out.

Reply:
I understand the reasoning and I will revert this change.

To comment on my reasoning here why I changed it in the first place: the Node.js Version here is mostly unrelated to the actual code used. TypeScript works with every LTS even long before it gets to be a LTS. So that won't be the problem. And the output code is depending on the `tsconfig` not on the used node version. So this change was mostly meant to use a current Node.js version which is officially supported and doesn't need changes in this config.
As I said, I understand your reasoning and will revert this.